samples/box_demo.cr

```diff
@@ -0,0 +1,41 @@
+# Boxes are invisible containers into which we can pack our widgets.
+# When packing widgets into a horizontal box, the
+# objects are inserted horizontally from left to right or right to left 
+# depending on whether Gtk::Box.pack_start() or Gtk::Box.pack_end() is used.
+# In a vertical box, widgets are packed from top to bottom or vice versa.
+# You may use any combination of boxes inside or beside other boxes to create the desired effect.
+
+require "gobject/gtk/autorun"
+
+window = Gtk::Window.new
+window.title = "Box demo!"
+window.connect "destroy", &->Gtk.main_quit
+window.border_width = 10
+
+box = Gtk::Box.new Gtk::Orientation::VERTICAL, spacing = 2

```

Comment:
We can use the shorthand enum reference syntax here and just say `Gtk::Box.new :vertical`.

Also `spacing = 2` is creating a local variable called `spacing` and then passing the result of the assignment expression as the second positional argument. To pass it as a keyword argument, we need to use `spacing: 2`